problem3.py

```diff
@@ -0,0 +1,16 @@
+#Time Complexity O(N)
```

Comment:
We can solve any linked list problem without using extra space mostly, so try to solve this without using extra space!